include/odp/api/spec/timer.h

```diff
@@ -36,16 +36,45 @@ extern "C" {
  */
 
 /**
- * Clock sources for timers in timer pool.
+ * Clock sources for timer pools
+ *
+ * ODP_CLOCK_DEFAULT is the default clock source and it is supported always. It is implementation
+ * defined which other clock sources are supported. See implementation documentation how the
+ * supported clock sources are mapped into these enumerations.
  */
 typedef enum {
-	/** Use CPU clock as clock source for timers */
-	ODP_CLOCK_CPU,
-	/** Use external clock as clock source for timers */
-	ODP_CLOCK_EXT
-	/* Platform dependent which other clock sources exist */
+	/** Clock source number 0 */
+	ODP_CLOCK_SRC_0,
+
+	/** Clock source number 1 */
+	ODP_CLOCK_SRC_1,
+
+	/** Clock source number 2 */
+	ODP_CLOCK_SRC_2,
+
+	/** Clock source number 3  */
+	ODP_CLOCK_SRC_3,
+
+	/** Clock source number 4  */
+	ODP_CLOCK_SRC_4,
+
+	/** Clock source number 5  */
+	ODP_CLOCK_SRC_5,
+
+	/** Number of clock source enumerations */
+	ODP_CLOCK_NUM_SRC
+
 } odp_timer_clk_src_t;
 
+/** The default clock source */
+#define ODP_CLOCK_DEFAULT ODP_CLOCK_SRC_0
+
+/** For backwards compatibility, ODP_CLOCK_CPU is synonym for ODP_CLOCK_DEFAULT */
+#define ODP_CLOCK_CPU ODP_CLOCK_DEFAULT
```

Comment:
ODP_CLOCK_CPU seems to be still used in examples and in test/performance after all commits of this PR. And ODP_CLOCK_EXT is still used in example/timer.

Reply:
I'll update all test apps once the API changes has been approved.

Reply:
Since this define is required only for backward compatibility you could also state that it will be deprecated in the future. Just to encourage people to use the new enumerations.